pkgs/applications/display-managers/ly/default.nix

```
@@ -1,20 +1,26 @@
-{ stdenv, lib, fetchFromGitHub, linux-pam }:
+{ stdenv, lib, fetchFromGitHub, linux-pam, git, libxcb, ncurses }:
+
+# This package is intended to be reworked, it is temporarily using a fork since
+# the original developer used a non-standard method of specifying .gitmodules.
+# https://github.com/nullgemm/ly/pull/279
 
 stdenv.mkDerivation rec {
-  pname = "ly";
-  version = "0.2.1";
+  name = "ly";
```

```suggestion
  pname = "ly";
  version = "";
```
version still needs to be set. unstable-XXXX-XX-XX would probably be most fitting here.